os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -284,7 +327,7 @@ static void _tizenrt_mutex_get(_mutex *plock)
 	int temp;
 	temp = sem_wait(*plock);
 	if (temp != 0) {
-		DBG_ERR("_tizenrt_spinlock failed!\n");
+		DBG_ERR(" failed!\n");
```

Comment:
How about to use only one DBG_ERR or ndbg? It's better to manage code.

Reply:
Will unifi all to DBG_ERR(), and DBG_INFO()